src/main/java/uk/gov/justice/digital/delius/data/api/StaffHuman.java

```diff
@@ -21,6 +21,10 @@
     private String forenames;
     @ApiModelProperty(value = "Family name", example = "Hancock")
     private String surname;
+    @ApiModelProperty(value = "Staff Grade", example = "PO")
+    private String staffGrade;
```

Comment:
Should we return a KeyValue for staff grade?  So they have both the code and the description?